### PR TITLE
Sayalis/feature 35 rate post

### PR DESCRIPTION
**Pull request by sayalis-cuelogic**
@onkartibe @aloka-cuelogic 

Card number #35
Card link https://trello.com/c/Vuuv1Obn

Completed rating for a post.

### CODE REVIEW

File: commentpost/static/commentpost/js/ratepost.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                post_id:$('#post_id').text()
+            },
+            success: function(response) {
+                if (response != 0) {
+                    $("#id_rate_post").addClass("hidden");
+                    $("#ratepost").text(response);
+                }
+            },
+            error: function(response) {
+                console.log("rating not doned")
```
Comment (aloka-cuelogic) [inline commentpost/static/commentpost/js/ratepost.js]: remove console

File: commentpost/static/commentpost/js/ratepost.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            },
+        });
+    });
+});
```
Comment (aloka-cuelogic) [inline commentpost/static/commentpost/js/ratepost.js]: add new line at EOF

**Comment by sayalis-cuelogic** [comment]
Worked on review comments.

File: commentpost/static/commentpost/js/ratepost.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+                post_rate: $('#id_rate_post').val(),
+                post_id:$('#post_id').text()
+            },
+            success: function(response) {
+                if (response != 0) {
+                    $("#id_rate_post").addClass("hidden");
+                    $("#ratepost").text(response);
+                }
+            },
+            error: function(response) {
```
Comment (aloka-cuelogic) [inline commentpost/static/commentpost/js/ratepost.js]: No Use of error function.
Not handling anything here

**Comment by sayalis-cuelogic** [comment]
Worked on review comments.
Changes done in js file for handling errors.

**Comment by aloka-cuelogic** [comment]
Reviewed PR

**Pull request merged**